variables.tf

```diff
@@ -47,6 +47,12 @@ variable "db_cluster_identifier" {
   description = "DB Cluster Identifier to associate with the IAM Role"
 }
 
+variable "lambda_log_retention_in_days" {
```

Comment:
This is an optional variable since it has a default value. 

Place it under `## OPTIONAL` heading. Currently, it is under 
`## REQUIRED`